parser/metadata.go

```diff
@@ -50,7 +51,10 @@ func readMetadata(metadata metadata, page *model.Page) {
 	})
 
 	readList(metadata["Tags"], func(val interface{}) {
-		page.Tags = append(page.Tags, val.(string))
+		page.Tags = append(page.Tags, model.Tag{
+			Name: val.(string),
+			Href: "/tags/" + strings.ToLower(val.(string)),
```

Comment:
You may declare an extra variable for the value so that there's only one type assertion with `.(string)`.